lib/TPP/PassBundles/LowLevelParallelization.cpp

```diff
@@ -62,14 +62,27 @@ struct LowLevelParallelization
     // that they are hoisted out of loops.
     pm.addPass(createCleanup());
 
-    mlir::tpp::SCFParallelLoopTilingOptions tilingOptions;
-    tilingOptions.tileSizes = parallelTaskGrid;
-    pm.addPass(createSCFParallelLoopTiling(tilingOptions));
+    mlir::tpp::LoopInsertionPassOptions loopInsertionPassOptions;
+    loopInsertionPassOptions.tileShapeM = tileShapeM;
+    loopInsertionPassOptions.tileShapeN = tileShapeN;
+    pm.addPass(createLoopInsertionPass(loopInsertionPassOptions));
+
+    mlir::tpp::LoopShufflePassOptions loopShufflePassOptions;
+    loopShufflePassOptions.shuffleOrder = shuffleOrder;
+    pm.addPass(createLoopShufflePass(loopShufflePassOptions));
+
+    mlir::tpp::LoopExpansionPassOptions loopExpansionPassOptions;
+    loopExpansionPassOptions.numOuterParallel = outerParallelLoops;
+    pm.addPass(createLoopExpansionPass(loopExpansionPassOptions));
 
     pm.addNestedPass<func::FuncOp>(createIntelAMXTileConfigInsertionPass());
     pm.addNestedPass<func::FuncOp>(createCanonicalizerPass());
     pm.addNestedPass<func::FuncOp>(createLoopInvariantCodeMotionPass());
     pm.addNestedPass<func::FuncOp>(createCanonicalizerPass());
     pm.addNestedPass<func::FuncOp>(createIntelAMXTileConfigHoistingPass());
+    pm.addPass(createCombineXsmmOpPass());
```

Comment:
These last passes need to go to a different "bundle" as they're not related to low level parallelization. But this can be done in a follow-up PR.